manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 1.38.107
+PackageLocale: en-US
+Publisher: Brave Software Inc.

```

```suggestion
Publisher: Brave Software Inc
```
This must match Control Panel.